### PR TITLE
Updated the README with factory instructions

### PR DESCRIPTION
**Pull request by ipbabble**
I have no idea what happened the changes I made. But I've added back in the instructions for adding factory cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,8 +122,70 @@ service](https://console.redhat.com/openshift/create).
...
+
+   Select the `Import Cluster` option beside the highlighted Create Cluster button.
+
+   On the "Import an existing cluster" page, enter the cluster name and choose Kubeconfig as the "import mode". Copy the kubeconfig details into the space provided. 
+
+   Note: After pressing `Import` you will be prompted to copy a command that can be used on the factory cluster. INGORE this message. It will go away, and is only useful if something goes wrong and you need to manually import. 
+
+## Factory setup using `cm` tool
+
+1. If you don't already have the `cm` tool then build from [here](https://github.com/open-cluster-management/cm-cli)
```
Comment (mhjacks) [inline README.md:145]: "download from"?  I don't think we want to imply they have to build it (and a link to the repo might be read just that way)

Comment (ipbabble) [reply README.md:145]: I'm not sure they have a download for this yet. Let me check

Comment (ipbabble) [reply README.md:145]: Funny I was about literally going to sit down and check in with them and read this first.

File: README.md [hunk 2]
```diff
@@ -82,8 +82,9 @@ service](https://console.redhat.com/openshift/create).
 1. Check the operators have been installed
 
    ```
-   UI -> Installed Operators
+   OpenShift UI -> Installed Operators
    ```
+   It takes time for all the operators and other components to be installed. It is useful to go back and forth between the OpenShift GitOps UI (see the following two steps) and the OpenShift UI to watch as applications and oeprators come up.
```
Comment (css-qe) [inline README.md:87]: typo: oeprators

File: README.md [hunk 3]
```diff
@@ -121,8 +122,70 @@ service](https://console.redhat.com/openshift/create).
...
 
 
-1. Check all applications are synchronised
+1. Check all applications are synchronized
 
+# Factory (Edge) setup
+
+It's time to connect a "remote" factory cluster to the data center (hub).
+
+## Factory setup using the ACM UI
```
Comment (css-qe) [inline README.md:131]: We might want to describe how to get the route for the ACM UI and how to log in (i.e., using the kubeadmin-password for hub)

File: README.md [hunk 4]
```diff
@@ -121,8 +122,70 @@ service](https://console.redhat.com/openshift/create).
...
+
+1. The hub will push down an ACM agent to the factory (edge).
+
+1. Once the agent is installed it will install OpenShift GitOps.
+
+1. OpenShift GitOps takes over and installs everything else. 
+ 
+You can check your Installed Operators on your factory (edge) OpenShift cluster and wait to see the OpenShift GitOps operator install. If you're really impatient, you can also check Workloads for the  `open-cluster-management-agent`  project to appear. This happens before OpenShift GitOps is installed.  
+
+When OpenShift GitOps installed and ready, launch the OpenShift GitOps (ArgoCD) console from up at the top right of the OpenShift console.
```
Comment (css-qe) [inline README.md:188]: When OpenShift GitOps is installed and ready

**Pull request closed without merging**